monitoring-server/cmake/Project.cmake

```diff
@@ -7,6 +7,7 @@ target_sources(Core
         include/core/comms/Packet.h
         include/core/events/MessageDelivery.h
         include/core/events/StopServer.h
+        include/core/events/TimerTrigger.h
```

Comment:
Please refer to the first comment in "monit/core: Create Device class definition" commit for similar question.

Reply:
Same as above